sftp/sftp.go

```diff
@@ -812,14 +812,16 @@ func (o *Object) SetModTime(modTime time.Time) error {
 	if err != nil {
 		return errors.Wrap(err, "SetModTime")
 	}
-	err = c.sftpClient.Chtimes(o.path(), modTime, modTime)
-	o.fs.putSftpConnection(&c, err)
-	if err != nil {
-		return errors.Wrap(err, "SetModTime failed")
+	if fs.ConfigFileGetBool(o.fs.name, "set_modtime", true) {
```

Comment:
I normally cache these things in the Fs struct in the NewFs function so the config file machinery doesn't get consulted on each operation.  I note that we did that with the `hashCheckDisabled` too... Fancy sending a follow up PR to fix those two things?  I'm going to merge this as-is though.